src/commands/fun/whois.js

```diff
@@ -59,37 +61,37 @@ export async function execute(interaction, client) {
     // Get joined date for member
     let joinDate = Math.floor(member.guild.joinedTimestamp / 1000);
     // Get user account create date
-    let createDate = Math.floor(user.createdAt / 1000);
+    let createDate = Math.floor(member.user.createdAt / 1000);
 
     // User Flags
     const userFlags = (await member.user.fetchFlags()).toArray();
 
     // Key Permissions
     const keyPermissions = member.permissions.toArray().filter((p) => elevatedPerms.includes(p));
-    if (keyPermissions.includes('ADMINISTRATOR'))
+    if (keyPermissions.includes('Administrator'))
```

Comment:
Hmm this is something I would be careful with concerning typos. I think you are all good on the typos part, just saying.
Maybe use a constant for this to make sure you get warnings in your favorite IDE:

``` 
const Permissions = {
   ADMINISTRATOR: "Administrator"
   etc: "etc"
};

// usage: Permissions.ADMINISTRATOR
```

Thoughts?

Reply:
Improvement --> own issue?
Perhaps there are named constants already in Discort.JS. Haven't checked